splunk_add_on_ucc_framework/commands/rest_builder/endpoint/base.py

```diff
@@ -33,10 +33,14 @@
 class RestEntityBuilder:
     _title_template = "[{}]"
     _rh_template = """
+special_fields = [
```

Comment:
we can, but how do we benefit from it? I think it's not worth adding extra logic and template just to avoid declaring an empty list. What do you think about it?